builtin/logical/pki/cert_util.go

```diff
@@ -1537,24 +1607,34 @@ func handleOtherSANs(in *x509.Certificate, sans map[string][]string) error {
 	return nil
 }
 
+const (
+	nameTypeEmail = 1
+	nameTypeDNS   = 2
+	nameTypeURI   = 6
+	nameTypeIP    = 8
```

Comment:
Previously this was `7`, should it still be?

Reply:
Fixed.